scripts/aclshow

```diff
@@ -169,8 +169,13 @@ class AclStat(object):
                     self.get_counter_value(rule_key, 'packets') == 'N/A'):
                 continue
             rule = self.acl_rules[rule_key]
+            rule_priority = -1
+            if "PRIORITY" in rule:
+                rule_priority = rule['PRIORITY']
+            elif "priority" in rule:
+                rule_priority = rule['priority']
```

Comment:
what if priority is 'pRiOrItY' ?

Reply:
addressed this code review comment in the next commit.